### DISCUSSION TITLE
Decentra-Network Core 0.1.0-alpha

### DISCUSSION THREAD

**Discussion by onuratakan** [Naruno/Naruno#134] [Announcements]
For the release notes please see the git repository:

https://github.com/Decentra-Network/Decentra-Network/blob/master/doc/release-notes/release-notes-0.1.0-alpha.md

<hr /><em>This discussion was created from the release <a href='https://github.com/Decentra-Network/Decentra-Network/releases/tag/v0.1.0-alpha'>Decentra-Network Core 0.1.0-alpha</a>.</em>

**Comment by cpyberry** [comment]
Congratulations on our first pre-release!
I am honored to be published in the release note.
